llvm/test/Instrumentation/TypeSanitizer/access-with-offset.ll

```diff
@@ -1,7 +1,8 @@
-; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --check-globals all --version 6
-; RUN: opt -passes='tysan' -S %s | FileCheck %s
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 6
```

Comment:
Should keep `--check-globals=all` to make sure chec lines for globals are (re)generated as needed?